Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -192,34 +187,42 @@ class ToolbarButton: UIButton {
             self.imageView?.tintColor = self.tintColor
         }
     }
+    
+}
+
+extension ToolbarButton: Themeable {
+    func applyTheme(_ themeName: String) {
+        guard let theme = ToolbarButton.Themes[themeName] else {
+            log.error("Unable to apply unknown theme \(themeName)")
+            return
+        }
+        selectedTintColor = theme.highlightButtonColor
+        disabledTintColor = theme.disabledButtonColor
+        unselectedTintColor = theme.buttonTintColor
+        self.tintColor = isEnabled ? unselectedTintColor : disabledTintColor
+        self.imageView?.tintColor = self.tintColor
+    }
 }
 
 class TabToolbar: Toolbar, TabToolbarProtocol {
```

Comment:
_Side note: if this was called `BottomToolbar` I would find this code so much easier to read._